src/App.js

```diff
@@ -15,12 +15,12 @@ const App = () => {
   }, [history]);
 
   return (
-    <div className="App">
+    <main className="App" role="main">
```

Comment:
I don't think we need to specify `role="main"` if semantically this is already `main`.
```suggestion
    <main className="App">
```

[Reference](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html)